index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/style.css">
+    <title>Resume</title>
+</head>
+<body>
+    <div style="width: 1024px; margin: 0 auto;"> 
+        <header style="display: flex; justify-content: space-between; align-items:center;">
+            <h1>Dzmitry Kuzmin resume</h1>
+            <img src="/images/photo-dima.jpg" alt="Kuzmin Dzmitry photo" width="130" style="margin-right:50px;" >  
+        </header>
+        <section>
+            <ol>
+                <li>Kuzmin Dzmitry.</li>
+                <li>Contact info:</li>
+                    <ul>
+                        <li>Tel.:+375(33)323-17-35</li>
+                        <li>E-mail:<a href="mailto:dimakuzmin9@gmail.com">dimakuzmin9@gmail.com</a></li>
+                    </ul>
+                <li>After working as a school psychologist, I decided that I didn't want to have anything
+                    with our political system. And along with this, there was a need for self-determination. 
+                    The desire to self-development and learn led me to the IT sphere.</li>
+                <li>I have no programming skills, but I have a great desire to become a competent specialist in this profession.</br> 
+                    Personal skills that I have and, in my opinion, will help me in further learning:</li>
+                    <ul>
+                        <li>easily teachable;</li>
+                        <li>responsibility;</li>
+                        <li>seIf-improvement;</li>
+                    </ul>
+                <li><code>I dont have expirience in JS.</code></li>
+                <li>I don't have any work experience in IT sphere. My last place work:</li>
+                    <ul>
+                        <li>Middle school No 9, Brest, Belarus. Position: Educational psychologist.</br>
+                            Duties and responsibilites:</li>
+                            <ul>
+                                <li>Psychological remidial activities with students;</li>
+                                <li>Prevention and diagnostics of psyhological and mental
+                                    health problems among students;</li>
+                                <li>Documentation menedgment;</li>
+                            </ul>
+                    </ul>
+                <li>Education:</li>
+                    <ul>
+                        <li>Bachelor's Degree in Social and Medical Psychology</br>
+                            2010-2016</br>
```

Comment:
`</ some tag>` is used when you close a tag, for example:  opening tag - `<li>`, closing tag - `</li>`. So you can use `<br>` (http://htmlbook.ru/html/br).